nautobot_dns_models/__init__.py

```diff
@@ -29,6 +29,13 @@
             ],
         },
     ],
+    "boolean": [
+        "django.forms.fields.BooleanField",
+        {
+            "widget": "django.forms.CheckboxInput",
+            "required": False,
+        },
+    ],
 }
```

Comment:
I believe you don't need to specify this. `boolean` is already included into Nautobot core as Checkbox